src/cli/commands/init.js

```diff
@@ -1,15 +1,17 @@
 'use strict'
 
+const fs = require('fs')
+const debug = require('debug')('ipfs:cli:init')
 const { ipfsPathHelp } = require('../utils')
 
 module.exports = {
-  command: 'init [config] [options]',
+  command: 'init [default-config] [options]',
   describe: 'Initialize a local IPFS node',
   builder (yargs) {
     return yargs
       .epilog(ipfsPathHelp)
-      .positional('config', {
-        describe: 'Node config, this should JSON and will be merged with the default config. Check https://github.com/ipfs/js-ipfs#optionsconfig',
+      .positional('default-config', {
```

Comment:
Whats the different between `init-config` and `default-config` other than one appears on `ipfs init` and the other on `ipfs daemon`?

Reply:
its the same thing, this just mirror go-ipfs

Reply:
I see.

Won't bikeshed then, although I'm curious if there was a reason in go-ipfs, @Stebalien ?

Reply:
Ah. The issue is that go-ipfs already _has_ a `--config` option (for specifying an out of repo config). It doesn't _work_ but it's already defined...

Reply:
Wait, actually, I'm confused. This is just a positional param so it doesn't really matter.

Reply:
Got it. The name does not matter but it would have been great if the name was the same. (no need to block )